tools/docs/advanced/using-multiple-main-topics.md

```diff
@@ -0,0 +1,48 @@
+# Using multiple main topics
+
+A consumer can also consume from multiple main topics. This can be done by passing a comma separated list of main topics to the `KAFKA_SOURCE_TOPICS` variable.
+
+For example, the following config:
+
+```
+KAFKA_SOURCE_TOPICS=price,product
+KAFKA_RETRY_INTERVALS=120
+KAFKA_GROUP=algolia
+```
+
+would generate the following topic chains:
+
+1. `price` -> `retry1.algolia.price` (delay of 120 secs) -> `deadLetter.algolia.price`
+1. `product` -> `retry1.algolia.product` (delay of 120 secs) -> `deadLetter.algolia.product`
```

Comment:
2.

Reply:
this is automatically incremented when displayed, see https://github.com/inviqa/kafka-consumer-go/blob/392a6c7cfd6748aebade50107e8961cd0fe2dabd/tools/docs/advanced/using-multiple-main-topics.md

Reply:
oh
![](https://media2.giphy.com/media/3o84U6421OOWegpQhq/giphy.gif)